src/sql/platform/theme/common/styler.ts

```diff
@@ -304,3 +305,17 @@ export function attachCheckboxStyler(widget: IThemable, themeService: IThemeServ
 		disabledCheckboxForeground: (style && style.disabledCheckboxForeground) || colors.disabledCheckboxForeground
 	}, widget);
 }
+
+export function attachInfoBoxStyler(widget: IThemable, themeService: IThemeService, style?: {
```

Comment:
Would be nice to define the type here like we do elsewhere in the file (IInfoBoxStyleOverrides)